docs/AuthRBAC.md

```diff
@@ -553,614 +555,582 @@ export const PostEdit = () => (
+import { TabbedForm } from '@react-admin/ra-rbac';
 
 const authProvider = {
     // ...
-    getPermissions: () => Promise.resolve([
-        { action: ['list', 'edit'], resource: 'products' },
-        { action: 'write', resource: 'products.reference' },
-        { action: 'write', resource: 'products.width' },
-        { action: 'write', resource: 'products.height' },
-        // 'products.description' is missing
-        { action: 'write', resource: 'products.thumbnail' },
-        // 'products.image' is missing
-        // note that the section with the name 'description' will be displayed 
-        { action: 'write', resource: 'products.section.description' },
-        // note that the section with the name 'images' will be displayed 
-        { action: 'write', resource: 'products.section.images' },
-        // 'products.panel.stock' is missing
-    ]),
+    getPermissions: () => Promise.resolve({
+        permissions: [
+            { action: ['list', 'edit'], resource: 'products' },
+            { action: 'write', resource: 'products.reference' },
+            { action: 'write', resource: 'products.width' },
+            { action: 'write', resource: 'products.height' },
+            // 'products.description' is missing
+            { action: 'write', resource: 'products.thumbnail' },
+            // 'products.image' is missing
+            // note that the tab with the name 'description' will be displayed 
+            { action: 'write', resource: 'products.tab.description' },
+            // note that the tab with the name 'images' will be displayed 
+            { action: 'write', resource: 'products.tab.images' },
+            // 'products.tab.stock' is missing
+        ],
+    }),
 };
 
 const ProductEdit = () => (
     <Edit>
-        <LongForm>
-            <LongForm.Section name="description" label="Description">
+        <TabbedForm>
+            <TabbedForm.Tab label="Description" name="description">
                 <TextInput source="reference" />
                 <TextInput source="width" />
                 <TextInput source="height" />
-                // not displayed
+                {/* not displayed */}
                 <TextInput source="description" />
-            </LongForm.Section>
-            <LongForm.Section name="images" label="Images">
-                // not displayed
+            </TabbedForm.Tab>
+            <TabbedForm.Tab label="Images" name="images">
+                {/* not displayed */}
                 <TextInput source="image" />
                 <TextInput source="thumbnail" />
-            </LongForm.Section>
-            // not displayed
-            <LongForm.Section name="stock" label="Stock">
+            </TabbedForm.Tab>
+            {/* not displayed */}
+            <TabbedForm.Tab label="Stock" name="stock">
                 <TextInput source="stock" />
-            </LongForm.Section>
-        </LongForm>
+            </TabbedForm.Tab>
+        </TabbedForm>
     </Edit>
 );
 ```
```

Comment:
This example seems to be the same as above with some mistakes. I suggest to remove it

```suggestion

```